packages/frontend/src/Views/Home/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { useReducer, useMemo } from "react";
+import { PaginationContext } from "./reducer/pagination.context";
+import { initialState, reducer } from "./reducer/pagination.reducer";
+import { PaginationContextType } from "./reducer/types/pagination.types";
+
+export const Pagination: React.FC = ({ children }) => {
+  const [state, dispatch] = useReducer(reducer, initialState);
+
+  const value: PaginationContextType = useMemo(() => [state, dispatch], [state, dispatch]);
```

Comment:
Czy jak usuniesz z deps arrayu dispatch to eslint ci krzyczy o jego brak? Jeżeli nie to do uśnięcia dispatch z depsów.  
Bo nie powinien, gdyż powinien wiedzieć że on się nigdy nie zmieni, bo pochodzi od React.useReducer